lib/engines/content_block_manager/app/presenters/content_block_manager/confirmation_copy_presenter.rb

```diff
@@ -0,0 +1,55 @@
+module ContentBlockManager
+  class ConfirmationCopyPresenter
+    def self.present(content_block_edition)
+      if content_block_edition.scheduled_publication.present?
```

Comment:
I think this check needs to be based on the `param` that's been passed from the last page - otherwise if I go to edit an existing scheduled edition and choose to edit it and publish it now, the `scheduled_publication` value is still on the edition and it shows this copy instead of the 'published' copy.